bodhi/tests/server/services/test_releases.py

```diff
@@ -17,15 +17,19 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 from unittest import mock
 import os
+from datetime import datetime
 
 import webtest
+from bs4 import BeautifulSoup
```

Comment:
You should also probably add it to the `tests_require` list in `setup.py`, for coherence.

Reply:
fixed in this updated commit:

https://github.com/fedora-infra/bodhi/pull/3500/commits/2e45e91f18ca0609cc1b3e327434bfdabcecb91d